frontend/src/CollectionForm/CollectionForm.tsx

```diff
@@ -0,0 +1,170 @@
+
+  const [popup, setPopup] = useState<boolean>(false);
+
+  const styles = useStyles();
+
+  const handleAddMural = (addedMural: any) => {
+    if (!addedMural) return
+    if (muralsInCollection.some((currMural: any) => currMural.id === addedMural.id))
+      return
+
+    setMuralsInCollection((oldMurals: any) => [...oldMurals, addedMural]);
+  };
+
+  const handleSave = () => {
+    if (!title.length || !description.length) return;
+
+    axios.post('http://localhost:3000/collection', {
+      collection: {
+        name: title,
+        description: description
+      },
+      murals: muralsInCollection.map((mural: any) => mural.id)
+    })
+      .then(() => {
+        setPopup(true);
+        setTimeout(() => setPopup(false), 5000);
+      })
+      .catch((err: any) => console.log(err));
+  };
+
+  const handleCancel = () => {
+    console.log("cancel")
+  };
+
+  useEffect(() => {
+    axios.get('http://localhost:3000/mural')
+      .then((response) => {
+        if (response.data) setMurals(response.data.murals.rows)
+      })
+      .catch(err => console.log(err));
+  }, []);
+
+  useEffect(() => {
+    const filtered = murals.filter((mural: any) => {
+      return mural.name.toLowerCase().includes(muralQuery)
+    });
+    setMuralResults(filtered);
+  }, [muralQuery, murals]);
+
+  return (
+    <div>
+      <form
+        noValidate
+        autoComplete="off"
+        onSubmit={(e: SyntheticEvent) => { e.preventDefault(); handleSave() }}>
+        <div className={styles.flexContainer}>
+          <InputBase
+            className={`${styles.title} ${styles.field}`}
+            defaultValue="Name the collection"
+            onChange={(e: any) => setTitle(e.target.value)}
+            inputProps={{ 'aria-label': 'New collection title' }}
+            onClick={() => setEditingTitle(true)}
+            onBlur={() => setEditingTitle(false)}
+            onMouseEnter={() => setHoveringTitle(true)}
+            onMouseLeave={() => setHoveringTitle(false)}
+            endAdornment={!editingTitle && (
+              <InputAdornment position="start">
+                <EditIcon color={hoveringTitle ? "primary" : "action"} />
+              </InputAdornment>
+            )}
+          />
+          <InputBase
+            className={styles.field}
+            multiline
+            rows={4}
+            placeholder="Add a description"
+            onChange={(e: any) => setDescription(e.target.value)}
+            onClick={() => setEditingDesc(true)}
+            onBlur={() => setEditingDesc(false)}
+            onMouseEnter={() => setHoveringDesc(true)}
+            onMouseLeave={() => setHoveringDesc(false)}
+            endAdornment={!editingDesc && (
+              <InputAdornment position="start">
+                <EditIcon color={hoveringDesc ? "primary" : "action"} />
+              </InputAdornment>
+            )}
+          />
+          <div className={styles.field}>
+            <Autocomplete
+              freeSolo
```

Comment:
freeSolo lets a user input anything, correct? Shouldn't it be restricted to only the autocomplete options?